src/vue/fields/InputField.vue

```diff
@@ -23,6 +23,20 @@
       @blur="onBlur"
     >
 
+    <span
+      v-if="isPasswordSwitcherShown"
+      class="input-field__password-toggle"
+      :class="{
+        'input-field__password-toggle--autofill-white': whiteAutofill
+      }"
+      @click="togglePasswordDisplay"
```

Comment:
If you try to press `TAB` key on your keyboard you’ll see that the cursor jumped to a next interactable element (input, button, textarea, etc.). pls replace `span` with `button`